mainServiceController/src/main/java/org/venice/piazza/servicecontroller/messaging/handlers/ExecuteServiceHandler.java

```diff
@@ -60,9 +72,14 @@
 	private MongoAccessor accessor;
 	@Autowired
 	private PiazzaLogger coreLogger;
+	@Autowired
+	private UUIDFactory uuidFactory;
+	@Value("${SPACE}")
+	private String SPACE;
 
 	private RestTemplate template = new RestTemplate();
-	
+	private ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
Would recommend autowiring the ObjectMapper like all of the *Controller classes in the pz-gateway do, and also deleting the ExecuteServiceHandler.makeObjectMapper() method.